src/odemis/driver/smaract.py

```diff
@@ -3171,7 +3162,10 @@ def _doReference(self, future, axes):
                         # TODO: Raise some error here
 
                 # if referenced, move to the safe position (if requested)
-                all_axes_referenced = all([self.referenced._value[a] for a in self._axis_map.keys()])
+                all_axes_referenced = all(
+                    self.referenced._value[a] for a in self._axis_map.keys()
```

Comment:
You can actually also drop the `.keys()`. Then it would likely fit on a single line.